wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/CreateGroupConversationActionHandler.swift

```diff
@@ -140,6 +141,44 @@ final class CreateGroupConversationActionHandler: ActionHandler<CreateGroupConve
         case (403, "access-denied"):
             action.fail(with: .accessDenied)
 
+        case (409, _):
+            let errorInfo = response.errorInfo
+            guard
+                let payload = Payload.ErrorResponse(response),
+                let nonFederatingDomains = payload.non_federating_backends
+            else {
+                return action.fail(with: .unknown(
+                    code: errorInfo.status,
+                    label: errorInfo.label,
+                    message: errorInfo.message
+                ))
+            }
+
+            if nonFederatingDomains.isEmpty {
+                handleSuccessResponse(response, action: action)
+            } else {
+                action.fail(with: .nonFederatingDomains(Set(nonFederatingDomains)))
+            }
+
+        case (533, _):
```

Comment:
Same question as above

Reply:
we don't have a special label. Just:
533
```
{
  "unreachable_backends": ["a.com", "b.org"]
}
```